src/bond-sdk-cards.js

```diff
@@ -84,6 +84,56 @@ class BondCards {
     });
   }
 
+  /**
+   * @description Show card PIN data
+   * @param {String} cardId The unique ID used to identify a specific card.
+   * @param {String} identity The temporary identity token allowing this call.
+   * @param {String} authorization The temporary authorization token.
+   * @param {Boolean} [reset=false] Reset the PIN and view
+   * @param {String} [htmlWrapper="text"] The expected type of response data.
+   * 'image' is wrapped in an <img src='<revealed_data>'/> HTML tag. 'text'
+   * would be inserted into a <span> element inside the iframe.
+   * @param {String} htmlSelector A selector for the field/element where the
+   * iFrame will be placed.
+   * @param {Object} [css={}] An object of CSS rules to apply to the response.
+   * @return {Promise} Returns a Promise that, when fulfilled,
+   * will either return an iFrame with the appropriate data or an error.
+   */
+  showPIN({
+    cardId,
+    identity,
+    authorization,
+    reset = false,
+    htmlWrapper = "text",
+    htmlSelector,
+    css = {},
+  }) {
+
+    const requestParams = {
+      method: reset ? "PATCH" : "GET",
```

Comment:
@kirbpowell coupled with the fincore PR

Reply:
[288](https://github.com/bond-tech/baas_financial_core/pull/288) or [290](https://github.com/bond-tech/baas_financial_core/pull/290)

Reply:
@kirbpowell nominally 288 but now both